### PR TITLE
iTunes explicit tag extended to 'yes', 'no', or 'clean'

### LINKED ISSUE #46: Change iTunes explicit tag to be 'yes', 'no', or 'clean'
Hey thanks for making this. It really makes it easier to generate a podcast RSS feed.

One thing that I notice when I use the W3C feed validator (https://validator.w3.org/feed/) on the built XML is that the validator doesn't like `true` or `false` for the `itunes:explicit` tag. It would be nice if instead of a boolean, we could pass ` 'yes' | 'no' | 'clean'`, as suggested by the feed validator. Alternatively, the bool could be turned into 'yes' or 'no'.

Screen Shot 2022-01-03 at 9 20 00 AM

For now, I'm using the following work around to get my feed validated.
```javascript
const podcast = new Podcast(feedOptions)
// ...add items for episodes...
const rssXml = podcast
  .buildXml()
  .replace(/<itunes:explicit>false<\/itunes:explicit>/g, '<itunes:explicit>no</itunes:explicit>')
  .replace(/<itunes:explicit>true<\/itunes:explicit>/g, '<itunes:explicit>yes</itunes:explicit>')
```

### PR DESCRIPTION
**Pull request by alecsg77**
Fix #46

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class Podcast {
     });
 
     customElements.push({
-      "itunes:explicit": !!options.itunesExplicit,
+      "itunes:explicit": typeof options.itunesExplicit === "boolean" ? !!options.itunesExplicit : options.itunesExplicit,
```
Comment (JumpLink) [inline src/index.ts:116]: I think we should not support `true` or `false` anymore and instead convert `true` to `"yes"` and `false` to `"no"`

File: src/types/item-itunes.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ export interface ItemITunes {
   /** (iTunes specific) author of the podcast */
   itunesAuthor?: string;
   /** (iTunes specific) specifies if the podcast contains explicit content */
-  itunesExplicit?: boolean;
+  itunesExplicit?: boolean | ('yes' | 'no' | 'clean');
```
Comment (JumpLink) [inline src/types/item-itunes.ts:5]: Why do you not also use the new type `FeedItunesExplicit` here?

**Review by JumpLink — CHANGES_REQUESTED** [review]
Thank you for the PR! I still have a few comments, can you resolve them?

File: src/index.ts (hunk 1 above)
Comment (alecsg77) [reply src/index.ts:116]: Two reasons.
Primary: the official documentation says true|false
The other values suggested by the W3C validator are referring to old Apple specs, back in 2008
https://web.archive.org/web/20080726084241/http://www.apple.com/itunes/store/podcaststechspecs.html#explicit
I used two other validators, suggested by Apple here https://help.apple.com/itc/podcasts_connect/#/itcbaf351599, and both validators accept true|false|"yes"|"no"|"clean".
Secondary: no breaking changes.

File: src/types/item-itunes.ts (hunk 2 above)
Comment (alecsg77) [reply src/types/item-itunes.ts:5]: I was following what I thought was the convention.
My preferred solution is to create an `ItunesExplicit` type and use that in both places. Better name than `FeedItunesExplicit`

**Review by JumpLink — APPROVED** [review]

**Pull request merged**